merge.py

```diff
@@ -0,0 +1,161 @@
+    print Timer(
+        'merge_sort(best_case(100))',
+        'from __main__ import merge_sort, best_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for worst case of list of first 100 integers"
+    print Timer(
+        'merge_sort(worst_case(100))',
+        'from __main__ import merge_sort, worst_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for best case of list of first 1000 integers"
+    print Timer(
+        'merge_sort(best_case(1000))',
+        'from __main__ import merge_sort, best_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for worst case of list of first 1000 integers"
+    print Timer(
+        'merge_sort(worst_case(1000))',
+        'from __main__ import merge_sort, worst_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for best case of list of first 2000 integers"
+    print Timer(
+        'merge_sort(best_case(2000))',
+        'from __main__ import merge_sort, best_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for worst case of list of first 2000 integers"
+    print Timer(
+        'merge_sort(worst_case(2000))',
+        'from __main__ import merge_sort, worst_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for best case of list of first 3000 integers"
+    print Timer(
+        'merge_sort(best_case(3000))',
+        'from __main__ import merge_sort, best_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for worst case of list of first 3000 integers"
+    print Timer(
+        'merge_sort(worst_case(3000))',
+        'from __main__ import merge_sort, worst_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for best case of list of first 4000 integers"
+    print Timer(
+        'merge_sort(best_case(4000))',
+        'from __main__ import merge_sort, best_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for worst case of list of first 4000 integers"
+    print Timer(
+        'merge_sort(worst_case(4000))',
+        'from __main__ import merge_sort, worst_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for best case of list of first 5000 integers"
+    print Timer(
+        'merge_sort(best_case(5000))',
+        'from __main__ import merge_sort, best_case'
+    ).timeit(10) / 10
+
+    print
+    print "Time for worst case of list of first 5000 integers"
+    print Timer(
+        'merge_sort(worst_case(5000))',
+        'from __main__ import merge_sort, worst_case'
+    ).timeit(10) / 10
+
+    print "These tests approximately model O(n log(n)) behavior for all cases"
+    print "so the merge sort is unbiased of how well sorted any inputs are prior"
+    print "to a merge sort. The merge sort is ideal for large, poorly sorted data"
+    print "sets, as compared to insertion sort which is O(n2) in worst cases."
+    print "However merge sorts are more expensive with space as compared to"
+    print "other algorithms, according to Wikipedia"
+    print "https://en.wikipedia.org/wiki/Merge_sort"
```

Comment:
Wow.  I really appreciate all these tests.  And I like your printed output.  Good work.